src/interfaces/Config.ts

```diff
@@ -56,6 +56,14 @@ export interface IMeasuresStackConfig {
     stackMeasuresToPercent?: boolean;
 }
 
+export interface IGeoDataIndex {
+    color?: number;
+    location?: number;
+    segmentBy?: number;
+    size?: number;
+    tooltipText?: number;
+}
+
```

Comment:
Please consider to move this to `src/interfaces/GeoChart.ts`
My idea is that all new GeoChart interfaces should be in one place